blog/_posts/2024-08-07-eks-to-aks-series-part1.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Mastering the Move: EKS to AKS by Example"
+description: "Learn how to migrate Amazon EKS workloads to Azure AKS using KEDA and Karpenter in this comprehensive guide."
+date: 2050-08-24
+author: Kenneth Kilty
+categories: general
+---
+
+## Introduction
+
+Many companies use multiple clouds for their workloads. These companies need to accommodate the cloud preferences of their customers. Kubernetes plays a central role in multi-cloud workloads due to its ability to provide a consistent and portable environment across different cloud providers.
+
+We’re excited to announce the first in a new documentation series designed specifically for customers already using Amazon EKS, to help them replicate or migrate their workloads to AKS: [Replicate an AWS event-driven workflow (EDW) workload with KEDA and Karpenter in Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/aks/eks-edw-overview)
```

Comment:
"We're excited to announce" always triggers my marketing detector and I really think we want to avoid that vibe in the blog. I'd suggest sticking to more neutral/fact-based language and let the reader decide whether it's exciting. :)